packages/eds-demo/src/app/globals.css

```diff
@@ -22,11 +22,27 @@ h2,
 h3,
 h4,
 h5,
-h6 {
-  font-family: Equinor, sans-serif;
+h6,
+h1.type,
+h2.type,
+h3.type,
+h4.type,
+h5.type,
+h6.type {
+  font-family: Equinor, sans-serif !important;
 }
 
 button,
 a {
   cursor: pointer;
 }
+
+.app {
+  background-color: var(--eds-color-bg-canvas);
+  color: var(--eds-color-text-strong);
+}
+
+.top-bar,
+.sidebar {
+  background-color: var(--eds-color-bg-surface);
+}
```

Comment:
This global.css is shared in the layout and used on all pages. I believe this is a problem for the static and dynamic exercise because one of the tasks is to define these colours.